src/js/components/MosaicView.js

```diff
@@ -58,21 +89,22 @@ export default function MosaicView({ locationName, camera }) {
       )
       window.removeEventListener("channel", handleChannelEvent)
     }
-  })
+  }, [])
 
   return (
     <div className="viewsArea">
       <ul className="views">
         {views.map((view) => {
           return (
-            <li key={view.channel} className="view">
-              <ChannelView
-                locationName={locationName}
-                camera={camera}
-                view={view}
-                currentMeta={currentMeta}
-              />
-            </li>
+            <ChannelView
+              locationName={locationName}
+              camera={camera}
+              view={view}
+              currentMeta={currentMeta}
+              setHasFocus={setHasFocus}
+              isFocusable={isFocusable}
+              key={view.channel}
+            />
```

Comment:
I think I preffer the option where you set the list item (`<li>`) element outside the `ChannelView` component. Now you have to assume the `ChannelView` will be used always inside list elements (`<ul>` or `<ol>`) and that is not being specified anywhere.
If you are keeping this new way, then I recommend to rename the component to `ChannelViewListItem`.

Reply:
I'm going to keep the new way as I think the views will always be lists. Also, I like that the `flex` styling is applied directly to the parent `ul` and child `li` elements.